Twig/Extension/SonataAdminExtension.php

```diff
@@ -21,6 +21,9 @@
 /**
  * Class SonataAdminExtension.
  *
+ * The output and getValueFromFieldDescription methods are public so that they
+ * can be tested, but should not be relied on.
+ *
```

Comment:
@greg0ire The [`@internal` PHPDoc tag](https://phpdoc.org/docs/latest/references/phpdoc/tags/internal.html) could be used here.
> The `@internal` tag can be used as counterpart of the `@api` tag, indicating that the associated Structural Elements are used purely for the internal workings of this piece of software.